Source/SteveSloth/Private/MyEnemyAIDeadState.cpp

```diff
@@ -0,0 +1,34 @@
+/****************************************************************************************
+ * Copyright:
+ * Name: Guy Lapensee
+ * Script: ItemBaseClass
+ * Date: April 29, 2024
+ * Description: Enemy AI dead state for enemy base class
+ * TODO:
+ * Known Bugs:
+ ****************************************************************************************/
+
+
+#include "MyEnemyAIDeadState.h"
+
+MyEnemyAIDeadState::MyEnemyAIDeadState()
+{
+	isDead = false;
+}
+
+MyEnemyAIDeadState::~MyEnemyAIDeadState()
+{
+}
+
+void MyEnemyAIDeadState::DeadAnimation()
```

Comment:
1: This function is never called.
2: This function should execute some commands or have functionality to actual change the animation of the enemy to its dead animation.
3: you should code in a count down timer to start once the animation is done playing.
Theses are some of the things that i would put into this function